stitches/gridstitch_fx.py

```diff
@@ -99,23 +101,31 @@ def stitch_gridded(rp, dl, fl):
                   lat=lat,
                   lon=lon)
 
-    # Use a for loop to fill a dictionary to hold the
-    # stitched gridded data products.
-    data_dict = {}
+    # Use a for loop to go create and save a netcdf file
+    # for each of the variables.
+    f = []
     for i in range(0, len(var_names)):
+        data_dict = {}
         v = var_names[i]
         d = gridded_data[i]
         a = variable_info[i]
         data_dict[v] = (["time", "lat", 'lon'], d, a)
 
-    # Store all of the information into a xr data set, this is the final
-    # object we will want to return.
-    ds = xr.Dataset(
-        data_vars=data_dict,
-        coords=coords,
-        attrs={'target data': 'Not available until full pipeline in place',
-               'stitching_id': str(rp['stitching_id'].unique()),
-               'recipe': 'need to figure out how to add this '}
-    )
-
-    return ds
+        # Store all of the information into a xr data set, this is the final
+        # object we will want to return.
+        r = rp.reset_index(drop=True).to_string()
+        ds = xr.Dataset(
+            data_vars=data_dict,
+            coords=coords,
+            attrs={'target data': 'Not available until full pipeline in place',
+                   'stitching_id': str(rp['stitching_id'].unique()[0]),
+                   'recipe': r,
+                   'variable':v})
+
+        m = str(rp.source_id.unique()[0])
+        # TODO there should be some way to figure out how way control how to write it out!
+        fname = "./stitched_" + m + '_' + v + '_' + str(rp['stitching_id'].unique()[0]) + '.nc'
```

Comment:
I just had a thought about this - the stitching ID doesn't tell us what actual recipe is being followed since the recipe for each stitching id is a random draw. we probably need to at least include the `draw` id in the file names for now and probably need to figure out something a little more robust somehow